### PR TITLE
docs: link to full list of exports

### PR DESCRIPTION
**Pull request by stempler**
Replaces https://github.com/halestudio/hale/pull/1093

### CODE REVIEW

**Comment by JohannaOtt** [comment]
@stempler Why is there a new replacement PR? Is another review needed here or is the one in the old PR sufficient?

**Comment by stempler** [comment]
> Why is there a new replacement PR?

I created a new PR because (1) the suggestion to not use an external link was not implemented in the original PR and (2) to avoid any problems related to encoding changes.

> Is another review needed here or is the one in the old PR sufficient?

Do any of the objections remain that were there for the old PR?

File: doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html [hunk 1]
```diff
@@ -31,11 +31,10 @@ <h1>Save the transformation result</h1>
 		work based on a XML target schema.
 	</p>
 	<p>
-		Supported data export formats in <i>hale�studio</i> are:
+		Supported data export formats in <i>hale�studio</i> can be viewed on the following page:
 	</p>
 	<ul>
-		<li><a href="../../reference/export/xml_data.html">XML</a></li>
-		<li><a href="../../reference/export/gml_data.html">GML</a></li>
+		<li><a href="../../reference/export/export.html">Supported formats for Export</a></li>
```
Comment (JohannaOtt) [inline doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html:37]: Have we had a chance now to verify if that is working as expected? The discussion [here](https://github.com/halestudio/hale/pull/1093#discussion_r1447089800) does not seem to have come to a clear result if I understand it correctly.

Comment (stempler) [reply doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html:37]: I tested it locally and it works as expected.

**Review by JohannaOtt — APPROVED** [review]
LGTM

**Review by emanuelaepure10 — APPROVED** [review]

**Pull request merged**